### PR TITLE
DOC-625 update install rpk to include update

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://redpandadata.atlassian.net/browse/DOC-625
Review deadline: Oct 11

## Page previews
https://deploy-preview-801--redpanda-docs-preview.netlify.app/current/get-started/rpk-install/

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/get-started/pages/rpk-install.adoc [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+:description: pass:q[Install or update `rpk` to interact with Redpanda from the command line.]
 
-The `rpk` tool is a single binary application that provides a way to interact with your Redpanda clusters from the command line. For example, you can use `rpk` to:
+The `rpk` tool is a single binary application that provides a way to interact with your Redpanda clusters from the command line. For example, you can use `rpk` to do the following:
 
-* Monitor your cluster's health.
-* Set up access control lists (ACLs) and other security features.
-* Create topics, produce to topics, and consume from topics.
+* Monitor your cluster's health
+* Create topics, produce to topics, and consume from topics
```
Comment (Deflaimun) [inline modules/get-started/pages/rpk-install.adoc]:
```suggestion
* Create, produce, and consume from topics
```

This feels more natural to me. Feel free to dismiss if you don't agree

**Comment by Deflaimun** [comment]
Can we add a link somewhere to the available versions? Specially when we say "install a version other than the latest" 
Something like `Check the available versions in our [release page](https://github.com/redpanda-data/redpanda/releases)`

link : https://github.com/redpanda-data/redpanda/releases

**Review by Deflaimun — APPROVED** [review]
See suggestions. This is a good change

**Comment by micheleRP** [comment]
> Can we add a link somewhere to the available versions? Specially when we say "install a version other than the latest" Something like `Check the available versions in our [release page](https://github.com/redpanda-data/redpanda/releases)`
> 
> link : https://github.com/redpanda-data/redpanda/releases

Yes, thanks: added link in [Check rpk version](https://deploy-preview-801--redpanda-docs-preview.netlify.app/current/get-started/rpk-install/#check-rpk-version)

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**